scripts/benchmarks/output.py

```diff
@@ -112,30 +85,140 @@ def generate_summary_table(chart_data: dict[str, list[Result]]):
                 best_key = key
 
         # Generate the row with the best value highlighted
+        if options.verbose: print(f"Results: {results}")
         for key in chart_data.keys():
             if key in results:
-                value = results[key].value
+                intv = results[key].value
                 if key == best_key:
-                    row += f" <ins>{value}</ins> |"  # Highlight the best value
+                    l.row += f" <ins>{intv:3f}</ins> {results[key].unit} |"  # Highlight the best value
+                else:
+                    l.row += f" {intv:.3f} {results[key].unit} |"
+            else:
+                l.row += " - |"
+
+        if len(chart_data.keys()) == 2:
+            key0 = list(chart_data.keys())[0]
+            key1 = list(chart_data.keys())[1]
+            if (key0 in results) and (key1 in results):
+                v0 = results[key0].value
+                v1 = results[key1].value
+                diff = None
+                if v0 != 0 and results[key0].lower_is_better: 
+                    diff = v1/v0
+                elif v1 != 0 and not results[key0].lower_is_better:
+                    diff = v0/v1
+                    
+                if diff != None:
+                    l.row += f"{(diff * 100):.2f}%"
+                    l.diff = diff
+
+        output_detailed_list.append(l)
+
+
+    sorted_detailed_list = sorted(output_detailed_list, key=lambda x: (x.diff is not None, x.diff), reverse=True)
+
+    diff_values = [l.diff for l in sorted_detailed_list if l.diff is not None]
+
+    if len(diff_values) > 0:
+        max_diff = max(max(diff_values) - 1, 1 - min(diff_values))
+
+        for l in sorted_detailed_list:
+            if l.diff != None:
+                l.row += f" | {(l.diff - 1)*100:.2f}%"
+                epsilon = 0.005
```

Comment:
Make this a configurable variable, and set it at ~5% by default.
We clearly have some work to do to stabilize the benchmarks, but let's do that later.

Reply:
--epsilon added as an option, with 0.005 default